source/dotnet/wholesale-api/WebApi.IntegrationTests/WebApi/V3/SettlementReportTests.cs

```diff
@@ -91,4 +91,52 @@ public sealed class SettlementReportTests : WebApiTestBase
 
         actualContent.Should().Be(expectedContent);
     }
+
+    [Theory]
+    [InlineAutoMoqData]
+    public async Task HTTP_GET_V3_Download_ReturnsExpectedContent(
```

Comment:
How about this name as the test does not only test the content but also parts of the HTTP response?
```suggestion
    public async Task HTTP_GET_V3_Download_ReturnsExpectedReponse(
```
Or, alternatively split into different tests, where e.g. "ExpectedFileName" could be one of the tests.

Reply:
Split the test into two parts - content and headers.